changelog.json

```diff
@@ -17,6 +17,14 @@
       "[Fixed] Windows: Installer not completing or getting stuck in a loop - #1875 #1863",
       "[Fixed] Move the 'Forgot Password' link to fix the tab order of the sign in view - #2200"
     ],
+    "0.6.3-beta7": [],
+    "0.6.3-beta6": [],
+    "0.6.3-beta5": [],
+    "0.6.3-beta4": [],
+    "0.6.3-beta3": [],
+    "0.6.3-beta2": [],
+    "0.6.3-beta1": [],
+    "0.6.3-beta0": [],
```

Comment:
I think we should keep them. Those _are_ real beta releases. They won't be shown on the site as that only shows releases from the `production` channel.